client/orderbook/orderbook.go

```diff
@@ -196,48 +227,55 @@ func (ob *OrderBook) Reset(snapshot *msgjson.OrderBook) error {
 	ob.seq = snapshot.Seq
 	ob.seqMtx.Unlock()
 
-	ob.marketID = snapshot.MarketID
+	atomic.StoreUint64(&ob.feeRates.base, snapshot.BaseFeeRate)
+	atomic.StoreUint64(&ob.feeRates.quote, snapshot.QuoteFeeRate)
 
-	// Clear all orders, if any.
-	ob.orders = make(map[order.OrderID]*Order)
```

Comment:
Funny, we must have been (unintentionally?) avoiding a race with a higher level synchronization mechanism in the caller.